src/main/java/picard/vcf/AccumulateVariantCallingMetrics.java

```diff
@@ -84,9 +87,7 @@ protected int doWork() {
                     totalHetDepth += detailedMetrics.TOTAL_HET_DEPTH;
 
                     // add it to the list of metrics for that sample so that we can merge them later
-                    if (!sampleDetailsMap.containsKey(detailedMetrics.SAMPLE_ALIAS)) {
-                        sampleDetailsMap.put(detailedMetrics.SAMPLE_ALIAS, new ArrayList<>());
-                    }
+                    sampleDetailsMap.computeIfAbsent(detailedMetrics.SAMPLE_ALIAS, f -> new ArrayList<>());
```

Comment:
you can merge these two lines. computeIfAbsent will return the eventual object...so:

```
sampleDetailsMap.computeIfAbsent(detailedMetrics.SAMPLE_ALIAS, f -> new ArrayList<>())
.add(detailedMetrics);
```